config/dependencies/gateway-api/gateway/gateway.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: Gateway
+metadata:
+  labels:
+    istio: ingressgateway
+  name: istio-ingressgateway
```

Comment:
Does it have to be called "istio-*"? Now that the Gateway API resource is being used, it looks like Istio is an implementation detail that could perhaps be abstracted at this level. WDYT?

Reply:
In the addresses, the value `value: istio-ingressgateway.istio-system.svc.cluster.local` is a must because this deployment relies on istio ingress gateway. 

I agree that the name can be anything, like kuadrant-gateway. I would not say, though, that Istio is an implementation detail. I do not think that kuadrant can, or should, abstract entirely Istio. I see kuadrant more of a Istio extension rather than a abstraction layer. This is out of context :) But dropping here anyway.

Anyway, this is not part of the kuadrant-operator release. This file belongs to the testing environment setup configuration. So I think it is ok as it is.

Reply:
Fair enough. Thanks for explaining, @eguzki.